src/sa_web/static/js/routes.js

```diff
@@ -7,6 +7,11 @@ var Shareabouts = Shareabouts || {};
     routes: {
       '': 'viewMap',
       'place/new': 'newPlace',
+      'place/new/observation': 'newPlaceObservation',
+      'place/new/question': 'newPlaceQuestion',
+      'place/new/idea': 'newPlaceIdea',
+      'place/new/complaint': 'newPlaceComplaint',
+      'place/new/greenwall': 'newPlaceGreenwall',
```

Comment:
Likewise here, we should consider pushing these configs into the config.yml or flavor-specific code